src/main/kotlin/com/project/goms/domain/studentCouncil/usecase/OutingBanUseCase.kt

```diff
@@ -0,0 +1,36 @@
+package com.project.goms.domain.studentCouncil.usecase
+
+import com.project.goms.domain.studentCouncil.common.property.OutingStatusExpTimeProperties
+import com.project.goms.domain.studentCouncil.entity.OutingStatus
+import com.project.goms.domain.studentCouncil.entity.repository.OutingStatusRepository
+import com.project.goms.domain.studentCouncil.usecase.dto.OutingBanDto
+import com.project.goms.global.annotation.UseCaseWithTransaction
+import mu.KotlinLogging
+import java.util.*
+
+private val log = KotlinLogging.logger {}
+
+@UseCaseWithTransaction
+class OutingBanUseCase(
+    private val outingStatusRepository: OutingStatusRepository,
+    private val outingStatusExpTimeProperties: OutingStatusExpTimeProperties
+) {
+
+    fun execute(outingBanDto: OutingBanDto) {
+        when (outingBanDto.status) {
+            true -> {
+                val outingStatus = OutingStatus(
+                    outingStatusUUID = UUID.randomUUID(),
+                    expiredAt = outingStatusExpTimeProperties.expiredAt
+                )
+                log.info { "이번주 외출 불가능 ㅅㄱ" }
```

Comment:
로그 작업을 이렇게 하시면...ㅋㅋㅋ 
goms log들은 중괄호가 아니라 소괄호로 했었는데, 이것도 컨벤션을 지키면 좋을거 같아요.
```suggestion
                log.info("이번주는 외출 불가능 상태 입니다." )
```